spec.html

```diff
@@ -28892,8 +29022,14 @@ <h1>String.raw ( _template_, ..._substitutions_ )</h1>
     <!-- es6num="21.1.3" -->
     <emu-clause id="sec-properties-of-the-string-prototype-object">
       <h1>Properties of the String Prototype Object</h1>
-      <p>The String prototype object is the intrinsic object <dfn>%StringPrototype%</dfn>. The String prototype object is a String exotic object and has the internal methods specified for such objects. It has a [[StringData]] internal slot with the value *""*. It has a `length` property whose initial value is 0 and whose attributes are { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
-      <p>The value of the [[Prototype]] internal slot of the String prototype object is the intrinsic object %ObjectPrototype%.</p>
+      <p>The String prototype object:</p>
+      <ul>
+        <li>is the intrinsic object <dfn>%StringPrototype%</dfn>.</li>
+        <li>is a String exotic object and has the internal methods specified for such objects.</li>
+        <li>has a [[StringData]] internal slot whose value is *""*.</li>
```

Comment:
Note for later: should this use `*""*` or `the empty String`?

Reply:
The spec is far more inclined to say `the empty String` (or sometimes `the empty string`), maybe because (depending on the font) it might be unclear whether `""` has a space between the quotes or not.